test/boto_test.py

```diff
@@ -34,7 +34,7 @@ class BotoTest(unittest.TestCase):
 
     def _get_parser(self, args=[]):
         # Get the argparse namespace object with the given args
-        parser = krux.cli.get_parser()
+        parser = get_parser()
```

Comment:
-.5 to this.

importing into the local namespace seems convenient until you start working on bigger projects and start wondering "Which get_parser am I calling? The one from library Foo or library Bar?" Now you have to scroll all the way back to the imports to answer that question.

I prefer being explicit about where the call comes from.